--- shush-1.0.2/Makefile.in~	2004-03-08 02:57:25.000000000 +0100
+++ shush-1.0.2/Makefile.in	2006-11-06 21:16:23.000000000 +0100
@@ -17,10 +17,10 @@
 	@(cd src && $(MAKE) shush)
 
 install: shush
-	$(INSTALL) -d -m 0755 $(bindir)
-	$(INSTALL) -m 755 src/shush $(bindir)
-	$(INSTALL) -d -m 0755 $(mandir)/man1
-	$(INSTALL) -m 644 shush.1 $(mandir)/man1
+	$(INSTALL) -d -m 0755 $(DESTDIR)$(bindir)
+	$(INSTALL) -m 755 src/shush $(DESTDIR)$(bindir)
+	$(INSTALL) -d -m 0755 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 644 shush.1 $(DESTDIR)$(mandir)/man1
 	@echo ; echo "Done!"
 
 clean:
